src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -191,4 +194,33 @@ public void teleopInit() {
     public void autonomousExit() {
         CommandScheduler.getInstance().cancelAll();
     }
+
+    @Override
+    public void simulationInit() {
+        PhysicsSim physicsSim = PhysicsSim.getInstance();
+        // TODO Find more accurate values
```

Comment:
For drive, I've been using 6000 RPM as the top speed.  Seems odd that turret would run slower, as I don't expect there's a lot more torque needed from it.

Reply:
And highly recommend adding a comment describing what the numbers mean (RPM, ticks per rotation, conversion from  minutes to 100ms).

Reply:
I switched to using constants, which should clarify what's going on. (The place where the constant is calculated also has a comment explaining it.) I also added a comment on the parameters of PhysicsSim.add*().